dags/bq_events_to_amplitude.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'start_date': datetime.datetime(2019, 6, 27),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': False,
+    'retries': 2,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+environment = '{{ task.__class__.deploy_environment }}'
+
+dag_name = 'bq_events_to_amplitude'
+
+# This connection needs access to both BQ and GCS
+gcp_conn_id = 'google_cloud_derived_datasets'
+
+amplitude_s3_conn = 'amplitude_s3_conn'
+amplitude_s3_bucket = 'com-amplitude-vacuum-mozilla-vacuum-wup'
+
+with models.DAG(
+        dag_name,
+        schedule_interval='0 1 * * *',
+        default_args=default_args) as dag:
+
+    # TODO update view
+    # For now, we assume the view is already updated
+    # See https://github.com/mozilla/bigquery-etl/issues/218
+
+    # Overwrite the table with yesterday's data
+    table_name = 'fenix_events_amplitude_v1'
+    table_creation = bigquery_etl_query(
+        destination_table=table_name,
+        date_partition_parameter=None,
+        parameters=('submission_date' + ':DATE:{{ds}}',)
+    )
+
+    project_id = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id
+
+    gcs_bucket = 'moz-fx-data-amplitude-export'
```

Comment:
@haroldwoo we'll need to create a gcs bucket for use with this job

Reply:
which gcp project? We will eventually migrate from derived-datasets to the shared-prod project

Reply:
I'm not sure. AFAIK you all are in charge of deciding which projects to use and for what purposes. It would be simpler for this all to be in derived-datasets (or shared-prod when we migrate) because then the derived tables and exported TSV data will be in the same project.

Reply:
Oh, right now the project used is whichever is defined in `google_cloud_derived_datasets`